index.d.ts

```diff
@@ -1488,7 +1488,7 @@ declare module binaryen {
   interface IfInfo extends ExpressionInfo {
     condition: ExpressionRef;
     ifTrue: ExpressionRef;
-    ifFalse: ExpressionRef;
+    ifFalse: ExpressionRef | null;
```

Comment:
Iirc an unset `ExpressionRef` (pointer to an `Expression` in linear memory) is just the value `0`, but isn't amended to `null`, which would not be a `number`.